scripts/run-conformance.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+which ginkgo
+if [ $? -ne 0 ]; then
+	echo "Downloading ginkgo tool"
+	go install github.com/onsi/ginkgo/ginkgo
```

Comment:
I found this doesn't work for me on Centos-7.6 with golang version 1.13.  it works by using `go get -u github.com/onsi/ginkgo/ginkgo`.

Reply:
I try to check it to `go get` but get this error:
```
Downloading ginkgo tool
go: github.com/openshift/api@v3.9.1-0.20191213091414-3fbf6bcf78e8+incompatible: invalid pseudo-version: preceding tag (v3.9.0) not found
```

I added a comment on do it inside the make file.

Reply:
> I try to check it to `go get` but get this error:
> 
> ```
> Downloading ginkgo tool
> go: github.com/openshift/api@v3.9.1-0.20191213091414-3fbf6bcf78e8+incompatible: invalid pseudo-version: preceding tag (v3.9.0) not found
> ```
> 

I wonder what openshift api has to do with gingko...

Reply:
Ok, I found what the difference is. When running with a "downloaded from google" go, it has the default u/s go mod proxy enabled. In yum installed ones it does not, and that causes the error.

Now, given that we are vendoring all our dependencies, and `go get gingko` reads the `go.mod` file as a side effect, I think temporarily moving to a tmp folder can be a bearable workaround.